### PR TITLE
Fix Strapi utils with ts migration

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Fixes https://github.com/strapi/strapi/issues/17366

### CODE REVIEW

**Review by Convly — APPROVED** [review]

File: packages/core/utils/src/index.ts [hunk 1]
```diff
@@ -21,23 +21,74 @@ export {
...
+import { pipeAsync, mapAsync, reduceAsync, forEachAsync } from './async';
+import convertQueryParams from './convert-query-params';
+import importDefault from './import-default';
+import * as template from './template';
+import * as file from './file';
+import * as traverse from './traverse';
+import webhook from './webhook';
+import { isOperator, isOperatorOfType } from './operators';
+
+export = {
```
Comment (joshuaellis) [inline packages/core/utils/src/index.ts:45]:
```suggestion
export {
```

**Review by joshuaellis — CHANGES_REQUESTED** [review]

File: packages/core/utils/src/index.ts (hunk 1 above)
Comment (joshuaellis) [reply packages/core/utils/src/index.ts:45]: i do believe?

Comment (alexandrebodin) [reply packages/core/utils/src/index.ts:45]: nope that's the point 😂

Comment (joshuaellis) [reply packages/core/utils/src/index.ts:45]: oh so you're doing a default export?

**Pull request merged**